src/Bookings.js

```diff
@@ -1,18 +1,77 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [customerId, setCustomerId] = useState(null);
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [errorOccurred, setErrorOccurred] = useState({
+    ok: true,
+    status: "",
+    statusText: ""
+  });
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`)
+      .then(res => {
+        if (!res.ok) {
+          setErrorOccurred({
+            ok: res.ok,
+            status: res.status,
+            statusText: res.statusText
+          });
+          throw new Error(res);
+        } else {
+          return res.json();
+        }
+      })
+      .then(data => {
+        setBookings(data);
+        setData(data);
+        setIsLoading(false);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  }, []);
+
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    if (searchVal.trim() === "") {
+      setBookings(data);
+    } else {
+      const filterBookings = bookings.filter(element => {
```

Comment:
Instead of using `element` here, it might be clearer to use `booking`. This value is a `booking` from our `bookings` array! It makes it a bit more readable knowing what the value is 😄 .